scrapy_zyte_api/_params.py

```diff
@@ -15,22 +15,274 @@
+    },
+    "articleListOptions": {
+        "default": {},
+        "is_extract_type": False,  # Not on its own.
+        "requires_browser_rendering": False,
+        "changes_fingerprint": True,
+    },
+    "articleNavigation": {
+        "default": False,
+        "is_extract_type": True,
+        "requires_browser_rendering": False,
+        "changes_fingerprint": True,
+    },
+    "articleNavigationOptions": {
+        "default": {},
+        "is_extract_type": False,  # Not on its own.
+        "requires_browser_rendering": False,
+        "changes_fingerprint": True,
+    },
+    "jobPosting": {
+        "default": False,
+        "is_extract_type": True,
+        "requires_browser_rendering": False,
+        "changes_fingerprint": True,
+    },
+    "jobPostingOptions": {
+        "default": {},
+        "is_extract_type": False,  # Not on its own.
+        "requires_browser_rendering": False,
+        "changes_fingerprint": True,
+    },
+    "product": {
+        "default": False,
+        "is_extract_type": True,
+        "requires_browser_rendering": False,
+        "changes_fingerprint": True,
+    },
+    "productOptions": {
+        "default": {},
+        "is_extract_type": False,  # Not on its own.
+        "requires_browser_rendering": False,
+        "changes_fingerprint": True,
+    },
+    "productList": {
+        "default": False,
+        "is_extract_type": True,
+        "requires_browser_rendering": False,
+        "changes_fingerprint": True,
+    },
+    "productListOptions": {
+        "default": {},
+        "is_extract_type": False,  # Not on its own.
+        "requires_browser_rendering": False,
+        "changes_fingerprint": True,
+    },
+    "productNavigation": {
+        "default": False,
+        "is_extract_type": True,
+        "requires_browser_rendering": False,
+        "changes_fingerprint": True,
+    },
+    "productNavigationOptions": {
+        "default": {},
+        "is_extract_type": False,  # Not on its own.
+        "requires_browser_rendering": False,
+        "changes_fingerprint": True,
+    },
+    "geolocation": {
+        "default": None,
+        "is_extract_type": False,
+        "requires_browser_rendering": False,
+        "changes_fingerprint": True,
+    },
+    "javascript": {
+        "default": None,
+        "is_extract_type": False,
+        "requires_browser_rendering": False,  # Not on its own.
+        "changes_fingerprint": True,
+    },
+    "actions": {
+        "default": [],
+        "is_extract_type": False,
+        "requires_browser_rendering": False,  # Not on its own.
+        "changes_fingerprint": True,
+    },
+    "jobId": {
+        "default": None,
+        "is_extract_type": False,
+        "requires_browser_rendering": False,
+        "changes_fingerprint": False,
+    },
+    "echoData": {
+        "default": None,
+        "is_extract_type": False,
+        "requires_browser_rendering": False,
+        "changes_fingerprint": False,
```

Comment:
It seems this does change the fingerprint, because the response contains echoData as-is